x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.tsx

```diff
@@ -170,7 +173,12 @@ const AlertsTable: React.FunctionComponent<AlertsTableProps> = (props: AlertsTab
                 )}
                 {renderCustomActionsRow &&
                   alerts[visibleRowIndex] &&
-                  renderCustomActionsRow(alerts[visibleRowIndex], handleFlyoutAlert, props.id)}
+                  renderCustomActionsRow(
```

Comment:
It might be better that we change this to use named parameters, since it's the parameters are getting long now

Reply:
As I want to merge this asap to unblock siem I most probably gonna do it in another PR